_posts/2015-09-16-04-19-odataIncludeAnnotations.md

```diff
@@ -39,7 +39,7 @@ The response will have all annotations:
   "@Hello.World":"Hello World.",
   "PerId":2,
   "Property.BirthdayAnnotation@odata.type":"#DateTimeOffset",
-  "Age@Property.BirthdayAnnotation":"2010-01-02T00:00:00+08:00",
+  "@Property.BirthdayAnnotation":"2010-01-02T00:00:00+08:00",
```

Comment:
why remove Age?

Reply:
I think "@Property.BirthdayAnnotation" is the annotation name, and 'Age' is the typo. Isn't it?

---
In reply to: [196252720](https://github.com/OData/WebApi/pull/1486#discussion_r196252720) [](ancestors = 196252720)